### ISSUE TITLE
Problem with special chars

### ISSUE THREAD

**Issue by whSarah** [pavelk2/social-feed#164]
Really nice plugin but I'm having a problem with special chars. When I try to get instagram images for "#bücherliebe" there is no result. Changing it to "#b\u00fccherliebe" works. 
But this is just a workaround since I need to be able to fill in the accounts dynamically.
Has someone experienced a similar problem?

**Comment by HartLarsson** [comment]
if you use PHP you can convert the name to Unicode encoding dynamically passign it to the plugin.

**Comment by whSarah** [comment]
No, I'm using ColdFusion but there probably is a function for that too.
Thank you, I'll check.
